sys/net/routing/nhdp/iib_table.c

```diff
@@ -468,6 +609,18 @@ static void reset_link_set_entry(iib_link_set_entry_t *ls_entry, timex_t *now, u
     ls_entry->exp_time = timex_add(*now, v_time);
     ls_entry->nb_elt = NULL;
     ls_entry->last_status = IIB_LT_STATUS_UNKNOWN;
+    ls_entry->metric_in = NHDP_METRIC_UNKNOWN;
+    ls_entry->metric_out = NHDP_METRIC_UNKNOWN;
+#if (NHDP_METRIC == NHDP_LMT_DAT)
+    memset(ls_entry->dat_received, 0, NHDP_Q_MEM_LENGTH);
+    memset(ls_entry->dat_total, 0, NHDP_Q_MEM_LENGTH);
+    ls_entry->dat_time.microseconds = 0;
```

Comment:
No, but I used timex values to be able to use functionality like `vtimer_now` or `timex_add`. Of course there is great potential to save a lot of memory but that would be a complex optimization step.